content/en/docs/contribute/intermediate.md

```diff
@@ -109,13 +109,19 @@ more information about the responsibilities of reviewers and approvers, see
   [style guide](/docs/contribute/style/style-guide). Only people listed as
   approvers in the
   [`OWNERS`](https://github.com/kubernetes/website/blob/master/OWNERS) file can
-  approve a PR. To approve a PR, leave an `/approved` comment on the PR.
+  approve a PR. To approve a PR, leave an `/approve` comment on the PR.
```

Comment:
Should line 115 also be changed, `/approve`?

Reply:
I figured out the OWNERS, OWNERS_ALIASES files.
I will address the nits that I found in a follow on PR.
Thanks!
/approve